Swag/Info.yaml

```diff
@@ -0,0 +1,12 @@
+Info:
+  version: 1.0.0
```

Comment:
Should this be v1?

Reply:
Yes, I just checked. It can be "user-defined". I thought it isn't. :)
Good catch.. hahha.

Reply:
Done!